api/services/tools/workflow_tools_manage_service.py

```diff
@@ -32,7 +33,7 @@ def create_workflow_tool(
         description: str,
         parameters: list[dict],
         privacy_policy: str = "",
-        labels: list[str] = None,
+        labels: Optional[list[str]] = None,
```

Comment:
This parameter is unused, you can remove it.

Reply:
(ditto) Will remove the unused parameter in the follow-up PR, covering the method's signature and usages.